includes/class-edd-fees.php

```diff
@@ -107,6 +107,9 @@ public function add_fee( $args = array() ) {
 		// Sanitize the amount
 		$args['amount'] = edd_sanitize_amount( $args['amount'] );
 
+		// Force the amount to have 2 decimal places
+		$args['amount'] = number_format( (float) $args['amount'], 2, '.', '' );
```

Comment:
we should use the `edd_currency_decimal_filter()` method here. It'll make sure that sites using accuracy different than `2` will work.